editor/tsconfig.json

```diff
@@ -25,5 +25,6 @@
       "worker-imports": ["src/core/workers/worker-import-utils.ts"]
     }
   },
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules"],
+  "include": ["src", "test", ".eslintrc.js"]
```

Comment:
Why do we need this now?

Reply:
without it there is a TS error in the actual .eslintrc.js file, but it's not important nor it blocks anything (removed it for now)